Makefile

```diff
@@ -48,24 +48,28 @@ ci-travis:
 	@printf "$$(tput bold)*****************$$(tput sgr0)\n"
 	@printf "$$(tput bold)* CI: Libraries *$$(tput sgr0)\n"
 	@printf "$$(tput bold)*****************$$(tput sgr0)\n"
-	@cd libraries/tock-cells && CI=true cargo test
-	@cd libraries/tock-register-interface && CI=true cargo test
+	#TODO: Fix these, or remove this code dump
+	#@cd libraries/enum_primitive && CI=true RUSTFLAGS="-D warnings" cargo test
```

Comment:
@ppannuto I think those `errors` occur because we don't use generic integers (https://github.com/rust-num/num-traits). Thus, the traits `NumCast`, `AsPrimitive` and the related macros should be removed and it should be fine I guess.

IMHO,  we should replace this library with `proc macros` instead